parking_permits/views.py

```diff
@@ -550,7 +551,10 @@ def post(self, request, format=None):
                 logger.error(f"Subscription validation failed. Error = {e}")
                 return Response({"message": str(e)}, status=400)
 
-            permit_id = validated_subscription_data.get("value")
+            meta = validated_subscription_data.get("meta")
+            meta_item = get_meta_item(meta, "permitId")
+            permit_id = meta_item.get("value")
```

Comment:
What if `meta_item` is None?  Can this happen?

Reply:
Well this cannot happen as `SubscriptionValidator` already validates that case and throws `SubscriptionValidationError` if this happens. Anyway added now extra check for this `meta_item` for a better code maintainability.